core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -354,7 +354,7 @@ public FileStatus getFileStatus(Path path) throws IOException {
     return new FileStatus(fileStatus.getLength(), fileStatus.isFolder(),
         getReplica(fileStatus), fileStatus.getBlockSizeBytes(),
         fileStatus.getLastModificationTimeMs(),
-        fileStatus.getCreationTimeMs(), new FsPermission((short) fileStatus.getMode()),
+        fileStatus.getLastAccessTimeMs(), new FsPermission((short) fileStatus.getMode()),
```

Comment:
do we have to update any documentation now, saying this is now last accessed time?

Reply:
Not really. We never mention this is creation time in the first place. Hadoop interface has always refer to this parameter as last access time so we are good. Basically our Hadoop client now implement access time correctly.